tslint-config-angular/package.json

```diff
@@ -18,19 +18,19 @@
   },
   "homepage": "https://github.com/infinum/js-linters#readme",
   "devDependencies": {
-    "@angular/compiler": "^8.0.0",
-    "@angular/core": "^8.0.0",
-    "rxjs": "~6.4.0",
-    "tslint": "^5.16.0",
-    "typescript": "~3.4.3",
-    "zone.js": "~0.9.1"
+    "@angular/compiler": "^10.1.1",
+    "@angular/core": "^10.1.1",
+    "rxjs": "~6.6.3",
+    "tslint": "^6.1.3",
+    "typescript": "~4.0.2",
+    "zone.js": "~0.11.1"
   },
   "peerDependencies": {
-    "tslint": "^5.16.0"
+    "tslint": ">=6"
```

Comment:
This is basically the most important change because ng10 apps use tslint 6.x so we needed to update this peerDep to avoid peerDep errors. The rest of changes in the PR are mostly updating example to ng10 and updating the tslint preset's dependencies.